client/packages/common/src/ui/forms/JsonForms/useJsonForms.tsx

```diff
@@ -265,33 +154,49 @@ export const useJsonForms = (
     </Box>
   );
 
+  useEffect(() => {
```

Comment:
I prefer to have the `useEffect` calls last, just above any `return` - just find it useful when reviewing the JSX to know what is going to change. let's call it my little quirk..